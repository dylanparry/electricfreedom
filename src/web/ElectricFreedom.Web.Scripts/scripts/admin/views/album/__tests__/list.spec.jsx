import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';

import AlbumList from '../list';

const setup = () =>
{
  const wrapper = shallow(<AlbumList />);

  return {
    wrapper,
    snapshot: toJson(wrapper),
  };
};

describe('AlbumList', () =>
{
  test('render', () =>
  {
    const { snapshot } = setup();

    expect(snapshot).toMatchSnapshot();
  });
});
