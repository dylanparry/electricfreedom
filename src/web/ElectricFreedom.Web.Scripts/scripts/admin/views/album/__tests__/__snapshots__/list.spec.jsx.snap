// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlbumList render 1`] = `
<p>
  Album List
</p>
`;
